packages/ui/src/molecules/RadioGroup/useRadioOption.ts

```diff
@@ -0,0 +1,33 @@
+import type { ChangeEventHandler } from 'react'
+import { createContext, useContext } from 'react'
+
+export interface RadioGroupContext {
+  /**
+   * Name to link children by context.
+   */
+  name: string
+  /**
+   * Value of checked child.
+   */
+  selectedValue: string | number
```

Comment:
You forgot to make it optional 😃 

```suggestion
  selectedValue?: string | number
```